### PR TITLE
Test hard removal of one control-plane unit

### PR DESCRIPTION
**Pull request by addyess**
is it possible to hard-remove a machine and the k8s cluster survive to add a replacement?

### CODE REVIEW

File: tests/e2e/tests/test_clustering.py [hunk 1]
```diff
@@ -48,3 +48,23 @@ def test_worker_nodes(instances: List[harness.Instance]):
     join_cluster(joining_node, token)
 
     util.wait_until_k8s_ready(cluster_node, instances)
+
+
+@pytest.mark.node_count(4)
+def test_control_plane_replacement(instances: List[harness.Instance]):
+    cp1, cp2, cp3, cp_r = instances
```
Comment (eaudetcobello) [inline tests/e2e/tests/test_clustering.py:55]: What is a cp_r ? cp1, cp2, cp3 are obvious enough but I'm not sure cp_r is clear.

Comment (addyess) [reply tests/e2e/tests/test_clustering.py:55]: CPR is "Cardiopulmonary Resuscitation". 

but in this case it's "Control-Plane Replacement" -- the instance i started to replace the one i'm going to remove

**Comment by bschimke95** [comment]
@addyess should we close this PR?

**Comment by addyess** [comment]
@bschimke95 shouldn't it work?

**Comment by addyess** [comment]
closing even though this should work.

**Pull request closed without merging**